homeassistant/components/vacuum/__init__.py

```diff
@@ -258,9 +259,16 @@ async def async_send_command(self, command, params=None, **kwargs):
         )
 
 
+@dataclass
+class VacuumEntityDescription(ToggleEntityDescription):
+    """An class that describes vacuum entities."""
+
+
 class VacuumEntity(_BaseVacuum, ToggleEntity):
```

Comment:
This is the legacy vacuum class I think. The state `StateVacuumEntity` class is the new one.

Reply:
Confusing 😟